easybuild/easyconfigs/p/Platanus/Platanus-1.2.4-foss-2017a.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'MakeCp'
+
+name = 'Platanus'
+version = '1.2.4' 
+
+homepage = 'http://platanus.bio.titech.ac.jp/'
+description = """PLATform for Assembling NUcleotide Sequences"""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['http://platanus.bio.titech.ac.jp/?ddownload=150']
+sources = ['%(name)s_v%(version)s.tar.gz']
+
+# fix Makefile to support newer gcc version
+patches = ['Platanus_new_gcc.patch']
```

Comment:
no, although I think `-funroll-loops` would be enabled if you use `-O3` (`'opt': True` in `toolchainopts`)

If you think the others should be included too, you'll need to add them in the `buildopts` line (there may be a good reason for it)